Fix uninitialized return value and unknown error() function.
Upstream commit:b3f9563

--- lib/backstepping.c~	2008-07-16 16:54:40.000000000 +0200
+++ lib/backstepping.c	2014-08-19 15:54:22.500330498 +0200
@@ -1,6 +1,6 @@
 /**************************************************************
  **
- ** Copyright (c) 2004-2005 LAAS/CNRS 
+ ** Copyright (c) 2004-2005,2014 LAAS/CNRS 
  **
  ** Thierry Peynot - 2004-2005
  **
@@ -357,7 +357,7 @@
 imuAttitudeAngles(double  angles[2],   /* output: the attitude angles */
 		  double  gravityVect[3] )  /* Estimation of the gravity Vector */
 {
-  int imuErr;
+  int imuErr = IMU_OK;
   double pitch;
   double cosPitch;
 
@@ -385,7 +385,7 @@
     }
   else
     {
-      imuErr = error (ERR_IMU_OFFLIMITS);
+      imuErr = ERR_IMU_OFFLIMITS;
       printf("califeIMU: gravityVect[2]=0.0: Pitch is over limit !!\n" );
     }
   
